RNA-seq/01-qc_trim_quant.md

```diff
@@ -6,6 +6,8 @@ These samples come from a project ([`PRJNA178120`](https://www.ebi.ac.uk/ena/dat
 
 Here we will perform quality control checks, trimming, and estimate the transcript abundances for a single sample, SRR585570.
 
+![](figures/rna-seq_1.png)
```

Comment:
Should this go under the `2. fastp` header like you did for `3. Quanitification with Salmon`? What are your thoughts?

Reply:
Hm. I do like that idea for its parallel-ness, however I'm also concerned that that means the participants will get about halfway through this document before they have this figure to orient them.

Reply:
Would putting it both places be redundant and weird?

Reply:
I see your point. It would be redundant, however it may still be useful to put emphasis on which step the `fastp` function is executed in. Was there a reason why the pipeline did not start with `FastQC`, because that would be a good first figure to orient the participants, followed by the `fastp` figure.

Reply:
FastQC does not filter or process the data in anyway. It produces a quality control report. That’s the rationale to not include it in the same way.

Reply:
Got it! In that case, I see no harm in leaving it as is.